swagger/swagger-invocation/invocation-core/src/main/java/org/apache/servicecomb/swagger/invocation/exception/ExceptionToResponseConverter.java

```diff
@@ -22,5 +22,9 @@
 public interface ExceptionToResponseConverter<T extends Throwable> {
   Class<T> getExceptionClass();
 
+  default int getOrder() {
```

Comment:
interface definition default to 100,
so strange

Reply:
Fixed